LanguageFeatures/Extension-types/exhaustiveness_variable_A03_t01.dart

```diff
@@ -5,8 +5,8 @@
 /// @assertion Exhaustiveness of a variable pattern is determined by the static
 /// type of the corresponding variable.
 ///
-/// @description Check that an exhaustiveness of a variable pattern is
-/// determined by the static type of the corresponding variable.
+/// @description Check that a lifted space union of a variable pattern is a
+/// union of the static type of the corresponding variable.
```

Comment:
Sorry about nit picking one more time, but here's my take on articles:

We use `the X` to indicate a contract with the listener/reader that the set of phenomena described by `X` contains one specific exemplar that we've already agreed on knowing about and focusing on. We use `an X` to indicate that this `X` is new in the discourse.

Much like the distinction between a declaring occurrence of `c` like `var c = new C()` vs. referencing occurrences like `c.foo()`, except that we don't have "the" in Dart, so we need to introduce a named storage location (that is, a variable) in order to be able to refer to a specific phenomenon among the ones that we've introduced.

Natural language just accepts the ambiguity, to some extent: "There were two dogs in the room, a humongous barking mastiff and a sleeping chihuahua. The dog tried to eat me." Ambiguous? Maybe, but natural language users will complain if that's a problem. ;-)

So:

```suggestion
/// @description Check that the lifted space union of a variable pattern is
/// determined by the static type of the corresponding variable.
```

This means that we introduce _a_ variable pattern, then refer to _the_ space which is associated with that pattern (it's uniquely defined when we know the pattern), and finally talk about the value of that space union.

In this particular case I'd suggest saying "is determined by" rather than "is a union of" because we want to emphasize that each static type has a space union (uniquely determined by that type), and that space union is also the space union of the pattern. If we say "is a union of" then we suggest that there are several of those, and we can choose whichever one we want.